boards/arm/nrf51_pca10031/doc/nrf51_pca10031.rst

```diff
@@ -0,0 +1,149 @@
+.. _nrf51_pca10031:
+
+nRF51-PCA10031
+##############
+
+Overview
+********
+
+Zephyr applications use the nrf51_pca10031 board configuration to run on the
+nRF51 Dongle (PCA10031) hardware. It provides support for the Nordic Semiconductor
+nRF51822 ARM Cortex-M0 CPU and the following devices:
+
+* :abbr:`ADC (Analog to Digital Converter)`
+* CLOCK
+* FLASH
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`I2C (Inter-Integrated Circuit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`PWM (Pulse Width Modulation)`
+* RADIO (Bluetooth Low Energy)
+* :abbr:`RTC (nRF RTC System Clock)`
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UART (Universal asynchronous receiver-transmitter)`
+* :abbr:`WDT (Watchdog Timer)`
+
+.. figure:: img/nrf51_pca10031.jpg
+     :width: 442px
+     :align: center
+     :alt: nRF51 PCA10031 Dongle
+
+     nRF51 PCA10031 Dongle (Credit: Nordic Semi)
+
+More information about the board can be found at the
+`nRF51 Dongle website`_. The `Nordic Semiconductor Infocenter`_
+contains the processor's information and the datasheet.
+
+Hardware
+********
+
+nRF51 Dongle has two external oscillators. The frequency of
+the slow clock is 32.768 kHz. The frequency of the main clock
+is 16 MHz.
+
+Supported Features
+==================
+
+The nrf51_pca10031 board configuration supports the following nRF51
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| ADC       | on-chip    | adc                  |
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
```

Comment:
I reckon there is no PWM hardware on the nRF51.

Reply:
This is a copy from nrf51_pca10028, so if this is wrong we should also update the existing board documentation.